paasta_tools/setup_kubernetes_internal_crd.py

```diff
@@ -76,24 +80,28 @@
         },
         spec={
             "group": "paasta.yelp.com",
-            "versions": [{"name": "v1beta1", "served": True, "storage": True}],
+            "versions": [{
+                "name": "v1beta1",
```

Comment:
same as above: does this version not need to match the api_version above on line 73?